src/Exception.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Dhii\File\Finder;
+
+/**
+ * An exception that is related to file finding.
+ *
+ * @since [*next-version*]
+ */
+class Exception extends \Exception implements ExceptionInterface
```

Comment:
Doesn't prevent repetitive typing of the same stuff that's in the namespace already. But good point, and I will fix this.